tools/ci/backend

```diff
@@ -8,7 +8,7 @@ set -x
 
 ./tools/lint --groups=backend --skip=gitlint,mypy # gitlint disabled because flaky
 ./tools/test-tools
-# We need to pass a parallel level to test-backend because CircleCI's
+# We need to pass a parallel level to test-backend because GitHub Actions'
```

Comment:
This comment may not be accurate after such a find/replace.

Reply:
I tested it without 6 processes, and it ran with 2 process which is correct. https://github.com/amanagr/zulip/runs/2123540401#step:14:14